### PR TITLE
DO NOT MERGE! feat: first draft images

### PR DESCRIPTION
**Pull request by niekvanstaveren**
So I think the implementation in the org chart should not be too difficult. I have showed Christofer how it looks with an image and he will come up with a design for the cards containing an image.

image

### CODE REVIEW

**Comment by haxxmaxx** [comment]
ok I'll go ahead and merge this then

File: src/card/card.js [hunk 1]
```diff
@@ -23,7 +23,11 @@ export default (data, cardStyling, selectionObj) => {
   const topColor = colorUtils.getDarkColor(backgroundColor);
   const fontColor = getFontColor(cardStyling, backgroundColor);
   const attributes = data.attributes || {};
-  let html = `<div class="sn-org-card-title">${attributes.label || data.id}</div>`;
+  let html = '<div class="sn-org-card-content">';
+  if (attributes.imageURL) {
+    html += `<img class="sn-org-card-image" src="${attributes.imageURL}"/>`;
```
Comment (Caele) [inline src/card/card.js]: think the URL needs to be validated so there is no possibility for script injection.

File: src/card/card.js [hunk 2]
```diff
@@ -23,7 +23,11 @@ export default (data, cardStyling, selectionObj) => {
...
+  }
+  html += `<div class="sn-org-card-textarea"><div class="sn-org-card-title">${attributes.label || data.id}</div>`;
```
Comment (Caele) [inline src/card/card.js]: hmm, possible that these also should be escaped?
`='<script type="text/javascript">function() {alert.show("hey")}()</script>'`
Doesn't really do anything in the client (chrome) but feels a bit risky as I can see it inject the node.

File: src/card/card.js (hunk 1 above)
Comment (niekvanstaveren) [reply src/card/card.js]: Well that would be a big check I think. Also everyone can see which checks we do, but more important which checks we don't do

File: src/card/card.js (hunk 2 above)
Comment (Caele) [reply src/card/card.js]: can't get it to run any of my injected code, so might be fine. But still should probable add a html encoding to all of these.

File: src/card/card.js (hunk 1 above)
Comment (Caele) [reply src/card/card.js]: hmm, I was thinking a encode URL would do it, but googling doesn't give me any of that. 
Think we can look at how it is done in the T&I object, `web/assets/client/services/media-tool/formatters/link.js`

**Pull request closed without merging**